usr/src/lib/libzfs_core/common/libzfs_core.h

```diff
@@ -70,7 +70,8 @@ int lzc_get_holds(const char *, nvlist_t **);
 enum lzc_send_flags {
 	LZC_SEND_FLAG_EMBED_DATA = 1 << 0,
 	LZC_SEND_FLAG_LARGE_BLOCK = 1 << 1,
-	LZC_SEND_FLAG_COMPRESS = 1 << 2
+	LZC_SEND_FLAG_COMPRESS = 1 << 2,
+	LZC_SEND_FLAG_SIGINFO = 1 << 3
```

Comment:
Given that there's no kernel or libzfs_core component to the SIGINFO stuff, I don't think it makes sense for it to be a libzfs_core flag.  Libzfs already knows to do this based on the new member in sendflags_t, so I think you can just remove `LZC_SEND_FLAG_SIGINFO` (and the handling of it, which is effectively a no-op).

Reply:
Thought I needed to handle both versions of send-ioctl, but clearly I don't.